wrapper/telemetry-wrapper.js

```diff
@@ -363,8 +364,11 @@ window.TelemetryWrapper.go = function (params, element) {
       ));
       valuesArePercent = true;
     } else {
-      yLabel = evolutions[0].description + ' - medians'; // i18n?
-      valueses = evolutions.map(evo => evo.percentiles(50));
+      if (params.percentile) {
+        percentileLabel = ' - ' + params.percentile + 'th percentile'; // i18n?
+      }
+      yLabel = evolutions[0].description + percentileLabel;
+      valueses = evolutions.map(evo => evo.percentiles(params.percentile || 50));
```

Comment:
Default parameters are set in setDefaultParams towards the bottom of the file. You can map 50 to "medians" to have a friendly view.